edgelet/edgelet-http-mgmt/src/server/module/mod.rs

```diff
@@ -55,7 +56,16 @@ where
         Err(err) => return Err(Error::from(err.context(context))),
     };
 
-    let module_spec = match CoreModuleSpec::new(name, type_, config, env) {
+    let pull_policy = match spec
+        .pull_policy()
+        .map_or(Ok(PullPolicy::default()), |policy| {
+            PullPolicy::from_str(policy)
+        }) {
+        Ok(pull_policy) => pull_policy,
+        Err(_err) => return Err(Error::from(context)),
```

Comment:
Construct an error here that mentions what the value of `spec.pull_policy()` was that couldn't be parsed.

Alternatively add such an `InvalidPullPolicy(String)` variant to `edgelet_core::ErrorKind` and have `<PullPolicy as FromStr>::Error` be `edgelet_core::Error` instead of `()`, so that you can then `.context()` it here directly.

Reply:
Sounds good